client/stylesheets/base.less

```diff
@@ -2605,9 +2605,8 @@ a.github-fork {
 			text-overflow: ellipsis;
 			position: relative;
 			padding-left: 10px;
-			color: @secondary-font-color;
-			.calc(width,
-			~"100% - 45px");
+			//color: @secondary-font-color;
```

Comment:
Works for me. It definitely makes it super clear/intuitive to tell when a user is offline.